zppy/defaults/default.ini

```diff
@@ -57,8 +57,28 @@ plugins = force_list(default=list())
 qos = string(default="regular")
 # Reservation -- if you have access to a node reservation, specify it with this parameter.
 reservation = string(default="")
+# Use for e3sm_to_cmip and/or ilamb tasks.
+# Name of the grid used by the relevant `[ts]` atm subtask
+ts_atm_grid = string(default="180x360_aave")
+# Use for e3sm_to_cmip and/or ilamb tasks.
+# Name of the `[ts]` atm subtask to depend on
+ts_atm_subsection = string(default="")
+# Use for e3sm_to_cmip task (but NOT the ilamb task) -- you can either set this, or
+# both ts_atm_grid and ts_land_grid
+# Name of the grid used by the relevant `[ts]` task
+ts_grid = string(default="180x360_aave")
+# Use for e3sm_to_cmip and/or ilamb tasks.
+# Name of the grid used by the relevant `[ts]` land subtask
+ts_land_grid = string(default="180x360_aave")
+# Use for e3sm_to_cmip and/or ilamb tasks.
+# Name of the `[ts]` land subtask to depend on
+ts_land_subsection = string(default="")
```

Comment:
From what I understand, you're moving these parameters up a level in the configuration hierarchy so that they can be reused across each `e3sm_to_cmip` subtask, rather than configuring `ts_grid` differently for each subtask. The intention is to eliminate redundant code/configuration.

Is this correct? If so, it sounds reasonable to me.